### PR TITLE
Travelling Merchant's Steel Sallet

### PR DESCRIPTION
**Pull request by KasparoVy**
Adds a modkit that can make any non-space helmet look like a steel sallet.
Fluff item for @TravellingMerchant. Sprites by TravellingMerchant.

### NOTE: Only the modkit is to be spawned in with Trav Noble, not the actual helmet object.

```
/obj/item/device/fluff/merchant_sallet_modkit
item: A modkit that can make most helmets look like a steel sallet.
what it does: Makes non-space helmets look like a steel sallet that has four adjustment states (changed with UI button).
ckey: Travelling Merchant
character: Trav Noble
```

The steel sallet can be adjusted via a UI button and has four states:
- Visor and bevor
- Visor only
- Bevor only
- Neither visor nor bevor

salletingame

And will obscure the head as makes sense for each state.
A converted helmet inherits all of the properties of the helmet that was converted, minus vision augmentation (i.e. meson helmet vision, stuff like that) and the converted helmet's toggle function.

And yes, I did have to make it exactly 100 line diff (not really :wink:).

### CODE REVIEW

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -363,11 +363,58 @@
...
+		qdel(src)
+	else
+		to_chat(user, "<span class='warning'>You can't modify [target]!</span>")
+
 //////////////////////////////////
 //////////// Clothing ////////////
 //////////////////////////////////
 
-//////////// Gloves ////////////
+//////////// Gloves //////////////
```
Comment (taukausanake) [inline code/modules/customitems/item_defines.dm]: Nerd

Comment (KasparoVy) [inline code/modules/customitems/item_defines.dm]: :thinking: Damn. PR on hold until I resolve this gamebreaking issue.

**Comment by Fox-McCloud** [comment]
@TullyBurnalot @Necaladun @FreeStylaLT

**Comment by TravellingMerchant** [comment]
Thank you! <3

**Pull request merged**